### ISSUE TITLE
[table] keyboard navigation not working on row actions

### ISSUE THREAD

**Issue by bjornalm** [IBM/carbon-addons-iot-react#1217]
### Describe the bug
Keyboard navigation not working on row actions, the overflow menu is not visible when using tab and hitting space to open results in a menu that you cant navigate.

### To Reproduce
1. Tab to a row, the focus state of the overflow menu isn't visible.
2. Hit space it opens the overflow menu
3. With it open, I can't get into the overflow menu to select edit/delete/etc.

### Expected behavior
Same behaviour as in Carbon?

**Comment by bjornalm** [comment]
Not sure when this functionality broke. It looks like it happened in stages. I went as far back as start of jan 2020 to check but even at that time it was not possible to navigate or select the row action overflow menu items using the keyboard.
